virttest/vt_console.py

```diff
@@ -0,0 +1,128 @@
+import threading
+
+
+from . import remote
+
+
+class ConsoleError(Exception):
+    """A base class for console errors."""
+
+    def __init__(self, *args):
+        Exception.__init__(self, *args)
```

Comment:
Can Line10 and Line11 be removed like class ConsoleBusyError? Is there special purpose?